addons/base_setup/views/res_config_settings_views.xml

```diff
@@ -83,7 +83,7 @@
                                         <div class="content-group">
                                             <div class="mt16" groups="base.group_no_one">
                                                 <label for="external_report_layout_id" string="Layout" class="col-3 col-lg-3 o_light_label"/>
-                                                <field name="external_report_layout_id" domain="[('type','=', 'qweb')]" class="oe_inline"/>
+                                                <field name="external_report_layout_id" domain="[('type', '=', 'qweb'), ('name', '!=', 'external_layout')]" class="oe_inline"/>
```

Comment:
the list of available choice doesn't make much sense, even when removing external_layout.

I picked a random one and got another traceback. So, we need to fix that with the proper domain, IMO

Reply:
@qdp-odoo After more careful investigation: instead of modifying the domain, I added a condition that remove infinite recursion in the report views for `external_layout` and all views that made use of it. I believe that this is actually a better fix than using domains to "hide" faulty views hehe! Hopefully, this works for you